src/cosmic/cosm.rs

```diff
@@ -797,7 +797,7 @@ impl Cosm {
                 state.vy = tgt.vy * (1.0 - dltdt) - obs.vy;
                 state.vz = tgt.vz * (1.0 - dltdt) - obs.vz;
 
-                if correction == LightTimeCalc::Abberation {
+                if correction == LightTimeCalc::Aberration {
```

Comment:
Same comment

Reply:
Resolved in commit 96a6ad32a5d0dc19579112d430d2411efecbbf96.